src/test/cypress/integration/quiz-exercise/QuizExerciseParticipation.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { artemis } from '../../support/ArtemisTesting';
+import { generateUUID } from '../../support/utils';
+
+// Accounts
+const admin = artemis.users.getAdmin();
+const student = artemis.users.getStudentOne();
+
+// Requests
+const courseManagementRequest = artemis.requests.courseManagement;
+
+// Page objects
+const multipleChoiceQuiz = artemis.pageobjects.multipleChoiceQuiz;
+
+// Common primitives
+let uid: string;
+let courseName: string;
+let courseShortName: string;
+let quizExerciseName: string;
+
+describe('Quiz Exercise Management', () => {
+    let course: any;
+    let quizExercise: any;
+
+    before('Set up course', () => {
+        uid = generateUUID();
+        courseName = 'Cypress course' + uid;
+        courseShortName = 'cypress' + uid;
+        cy.login(admin);
+        courseManagementRequest.createCourse(courseName, courseShortName).then((response) => {
+            course = response.body;
+            courseManagementRequest.addStudentToCourse(course.id, student.username);
+        });
+    });
+
+    afterEach('Delete Quiz', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteQuizExercise(quizExercise.id);
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteCourse(course.id);
+    });
+
+    describe('Quiz exercise participation', () => {
+        beforeEach('Create quiz exercise', () => {
+            uid = generateUUID();
+            quizExerciseName = 'Cypress Quiz ' + uid;
```

Comment:
If I am not mistaken the `uid` is only used in the `before` hook to create the course and in the `beforeEach` hook to create the quiz name. The superfluous. I think now the uid can even be constant and only has to be defined in the before hook and doesn't need testwide scope.
```suggestion
            quizExerciseName = 'Cypress Quiz ' + generateUUID();
```

Reply:
addressed <a href="https://github.com/ls1intum/Artemis/pull/3969/">here</a>